site/en/docs/devtools/css/color/index.md

```diff
@@ -0,0 +1,157 @@
+
+## Color Picker elements {: #color-picker-elements }
+
+Here's a description of each of the UI elements of the **Color Picker**:
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/AzrBlIl6iPByojkmoU4y.png", alt="The Color Picker, annotated.", width="800", height="908" %}
+
+1. **Shades**.
+1. **Eyedropper**. See [Sample a color anywhere with the Eyedropper](#eyedropper).
+1. **Copy to clipboard**. Copy the **Display value** to your clipboard.
+1. **Display value**. Arguments of the chosen color space.
+1. **Contrast ratio**. Available only for `color` values. It's the difference between `color` and `background-color`. 
+1. **Color palette**. Click a square to change the color to that of the square.
+1. **Gamut boundary**. This line is available only for the new color spaces and the [`color()`](/articles/high-definition-css-color-guide/#the-color-function) function. They can produce both HD and non-HD colors. The line lets you distinguish between HD and non-HD.
+1. **Color circle**. Drag this circle across the shades to change the display value.
+1. **Hue slider**.
+1. **Opacity slider**.
+1. **Display value switcher**. Pick a color space from the drop-down list. See [Convert colors](#convert-colors).
+   {% Aside %}
+   Alternatively, to open the drop-down menu directly, hold down <kbd>Shift</kbd> and click on the color preview icon next to the color value.
+   {% endAside %}
+1. **Expand contrast ratio**. Opens the corresponding section that lets you [Fix contrast](#fix-contrast).
+1. **Color palette switcher**. Click it to toggle between:
+
+   - [**Material** Design palette](https://m2.material.io/design/color/the-color-system.html#color-usage-and-palettes).
+   - **Custom** palette. To manually add the current color to this palette, click {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/eY8MaTQqlXF3oiT6STmy.svg", alt="Add.", width="20", height="20" %}.
+   - **CSS Variables** palette. Lists all custom CSS variables (appended with `--`) on your page.
+   - **Page colors** palette. To generate this palette, DevTools looks for all the colors in your stylesheets.
+
+## Choose a color space {: #choose-color-space }
+
+To choose a color space:
+
+1. <kbd>Shift-click</kbd> the preview icon next a color value. A drop-down list opens.
+
+   {% Aside %}
+   Alternatively, click the **Display value switcher** in the **Color Picker**.
+   {% endAside %}
+
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/VDuxFzZRnRkSWsSfTlhx.png", alt="The drop-down list with all the supported color spaces.", width="800", height="838" %}
+
+1. Choose one of the following color spaces:
+
+   - [`#<hex-color>`](https://developer.mozilla.org/docs/Web/CSS/hex-color)
+   - [`rgb(X X X / .X)`](https://developer.mozilla.org/docs/Web/CSS/color_value/rgb).
+   - [`hsl(X X X / X)`](https://developer.mozilla.org/docs/Web/CSS/color_value/hsl).
+   - [`hwb(X X X / .X)`](https://developer.mozilla.org/docs/Web/CSS/color_value/hwb).
+   
+   Or one of the experimental ones:
```

Comment:
Are these color still experimental?

Reply:
In the MDN docs that I reference, they are.

Reply:
Replaced with new